target/linux/generic/pending-6.6/192-fix-libbpf-Wmaybe-uninitialized.patch

```diff
@@ -0,0 +1,58 @@
+From: Sam James <sam@gentoo.org>
```

Comment:
This patch looks like a backport from 6.12, so make it clear:

```shell
git mv \
  target/linux/generic/pending-6.6/192-fix-libbpf-Wmaybe-uninitialized.patch \
 target/linux/generic/backport-6.6/192-v6.12-fix-libbpf-Wmaybe-uninitialized.patch
```

Reply:
Applied suggested rename and force-pushed new commit.